docker-1_host/templates/docker-compose.yml

```
@@ -89,14 +89,13 @@ services:
     ulimits:
       nofile: 90032
   elasticsearch:
-    image: docker.elastic.co/elasticsearch/elasticsearch:6.2.4
+    image: docker.elastic.co/elasticsearch/elasticsearch:7.16.2
```

```suggestion
    image: docker.elastic.co/elasticsearch/elasticsearch:7.16.3
```

I believe there's no good reason to not use the latest 7.16. Please, do the same thing everywhere else.

---

Latest stable version used in https://github.com/QualityUnit/LiveAgent-Docker/pull/6/commits/02a2a4467d699217ef4ab827c163107434e35396